src/main/java/core/basesyntax/handler/BalanceOperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.handler;
+
+import core.basesyntax.db.Storage;
+
+public class BalanceOperationHandler implements ShopOperationHandler {
+    @Override
+    public void doOperation(String fruitName, String quantityToOperate) {
+        Integer oldQuantityValue = Storage.FRUIT_STORAGE.get(fruitName.toLowerCase());
+        Storage.FRUIT_STORAGE.put(fruitName.toLowerCase(),
+                Integer.parseInt(quantityToOperate) + oldQuantityValue);
```

Comment:
Balance handler should override previous quantity (it's like correction after revision):
```suggestion
        Storage.FRUIT_STORAGE.put(fruitName.toLowerCase(),
                Integer.parseInt(quantityToOperate));
```

Reply:
Done

Reply:
Done